reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+- Docs for scheduler internals, cluster-admin best practices; standardize triage
+  process
+
+**SIG Security**:
+- Future Tech Leads
+
+**SIG Storage**:
+- Reviewers
+- Issue triage (creating and running a program)
+- Feature work for things that are co-owned by sig-node, sig-apps, and 
+sig-scheduling (ContainerNotifier, Volume expansion for stateful set, and more)
+
+**SIG Testing**:
+- Many more companies to invest in this area heavily and bring steady 
+contributors to grow the contributor ladder in areas that are crucial to the projects
+  infrastructure
+
+**SIG UI**:
+- contributors who will stick around with AngularJS, golang, and knowledge of
+  Kubernetes client-go package
+
+**SIG Usability**:
+- We are currently working on a jobs to be done study and an effort to define
+  universal personas for the upstream project. Any one is welcome to join and
+  participate in these efforts, especially any user researchers, designers, and
+  new contributors.
+
+**SIG Windows**: 
+- e2e test coverage and API reviewers
+
+**WG K8s Infra**
+-fill this out 
+
+**WG Multitenancy** 
+- We have three main projects: Hierarchical Namespace, Virtual Cluster Project,
+  and Multi-tenancy Profiles (think conformancy but for secure multi-tenant
+  clusters). Contributors and interested parties welcome! 
+
+
+Check out the contributor guide for a comprehensive guide to getting started:
+https://k8s.de/guide.
+
+
+
+### Other types of upstream contributions:
+
+The above list is good for contributors who have the time to join a SIG and/or 
+have support from their employer to submit patches and other common activities
+upstream. Below are areas of the project that need help, without commits
+necssarily, and would help escalate the project. 
+
+- Comment on [KEPs][kep] with your use case (this is helpful from end users
+  too!)
+- Tag `sig/security` on Issues and PRs that you review and have security
+  concerns
+- [SIG Multicluster] needs use cases and validating our approaches for different
+  environments and deployment models 
+- [SIG Usability] would like more participants for their [Job Study] and many
+  other studies that are going on.
+- [SIG Contributor Experience] would welcome part-time and full time contributor
+  community managers; will mentor and grow dedicated contributors in a large
+  environment
+- [SIG Architecture] would like cluster operators to take a 
+[production readiness survey]
+
+
+[SIG Multicluster]: https://git.k8s.io/community/sig-multicluster
+[SIG Usability]: https://git.k8s.io/community/sig-usability
+[Job Study]: https://docs.google.com/document/d/1lkPQdBEw-Xb5GEZ48WnpBgQdZ01EmTBhjcxJBlu5qJs/edit#heading=h.xsg4f6e6yk0p
+[SIG Contributor Experience]: https://git.k8s.io/community/sig-contributor-experience
+[SIG Architecture]: https://git.k8s.io/community/sig-architecture
+[production readiness survey]: https://docs.google.com/forms/d/e/1FAIpQLSc-J-Ydu5vp5G9vdvV5gBcraEDN_Bl-HSkVm15vAlU_orDvoA/viewform
+
+
+## Growth Areas
```

Comment:
This section should be above `/help-wanted` for sure, and probably directly under `Accolades`.